common/source/docs/common-msp-osd-overview.rst

```diff
@@ -60,18 +71,18 @@ Panel Items Currently Supported
 
 
 OSD Panel Item Configuration
-============================
+----------------------------
 
 Each OSD panel item uses a set of three variables to be set: 
 
-- ``OSDn_ITEM_ENABLE`` - activates the respective item when set to 1.
-- ``OSDn_ITEM_X`` and ``OSDn_ITEM_Y`` set the horizontal and vertical position of the item, starting with ``X = 0`` and ``Y = 0`` in the upper left corner of your screen. 
+- ``OSDn_<ITEM>_ENA`` - activates the respective item when set to 1.
```

Comment:
OSDn<ITEM>_EN

also should give an example :ref:\`OSD2_BAT_BOLT_EN<OSD2_BAT_VOLT_EN>\`